klaytn/README.md

```diff
@@ -0,0 +1,38 @@
+# Chainlink External Adapter for Klaytn
+
+This adapter is built to fulfill Chainlink oracle requests and send transactions to a [Klaytn](https://github.com/klaytn/klaytn) node.
+
+### Environment Variables
+
+| Required? |    Name     |                                             Description                                              | Options |       Defaults to       |
+| :-------: | :---------: | :--------------------------------------------------------------------------------------------------: | :-----: | :---------------------: |
+|           |     URL     |                            A URL to a JSON-RPC (HTTP RPC) node on Klaytn                             |         | `http://localhost:8551` |
+|    ✅     | PRIVATE_KEY | The private key to sign transactions with. Must have fulfillment permissions on the Oracle contract. |         |                         |
+
+---
+
+### Input Parameters
+
+| Required? |        Name         |                                 Description                                  | Options | Defaults to |
+| :-------: | :-----------------: | :--------------------------------------------------------------------------: | :-----: | :---------: |
+|    ✅     |      `address`      |                The oracle contract to fulfill the request on                 |         |             |
+|    ✅     |       `data`        | The data that contains functionSelector and dataPrefix prefix in the request |         |             |
+|    ✅     |      `topics`       |                      The fulfillment function selector                       |         |             |
+|    ✅     | `result` or `value` |                    The value to fulfill the request with                     |         |             |
+
+---
+
+### Sample Input
```

Comment:
Added a space for something like
```suggestion
```json
{
  "jobRunID": "1",
  "data": {
             "address": '0xf1e73a60687affcb1d6308fcb2ef1da3865ff797',
            "data":
        "0x000000000000000000000000344d20e71a76b5a44fff62438cd17446a331bf3e70ae84788a565ad436a112ee40d236e6f95106e22bd45cd8b78c8fe8c93c4df0000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000344d20e71a76b5a44fff62438cd17446a331bf3ed204b3fb0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000603d9865000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000",
            "topics": [
              "0xd8d7ecc4800d25fa53ce0372f13a416d98907a7ef3d8d3bdd79cf4fe75529c65",
              "0x3836643062336366366262313466306338633237363234316636383361363765",
            ],
            "value": "1003904"
  }
}
```
```

Pending that comment about that value param.